scripts/settings.yaml

```diff
@@ -0,0 +1,511 @@
+    be output.
+  Notes: For instance, if the solution frequency is 10 hz, and the "output every n
+    obs" parameter is 2, it means that the observation output will occur at a rate
+    of 5hz.  Since the observations are the information used by the Piksi receiving
+    corrections from the connected Piksi, this determines the rate of information
+    sharing for RTK solution output. This parameter is designed to tune the rate at
+    which correction information is passed from one Piksi to the other as to efficiently
+    use radio modem bandwidth and fit with user applications.
+- group: solution
+  name: dgnss_solution_mode
+  type: enum
+  units:
+  default value:
+  enumerated possible values: Low Latency, Time Matched
+  Description: Determines the type of RTK solution which will be output.
+  Notes: A low latency solution uses an internal model of anticipated satellite observations
+    to provide RTK output with minimal latency but slightly reduced accuracy.  This
+    mode should be used for most applications.  Where accuracy is desired with the
+    expectation of increased latency, the "Time matched" mode can be chosen.  This
+    means that the RTK output will require a corresponding set of correction observations
+    for each timestamp.
+- group: solution
+  name: dgnss_filter
+  type: enum
+  units:
+  default value: Fixed
+  enumerated possible values: Fixed,Float
+  Description: Determines the type of IAR the pikis will attempt to achieve.
+  Notes: If "fixed", the Piksi will output a fixed integer ambiguity estimate.  If
+    no fixed solution is available, it will rever to the float solution.  If "float",
+    the device will only output the float integer ambiguity estimate.
+- group: solution
+  name: known_baseline_n
+  type: double
+  units: meters (north)
+  default value: '0'
+  enumerated possible values:
+  Description: Determines the baseline vector for the "init known baseline" feature.
+  Notes:
+- group: solution
+  name: known_baseline_e
+  type: double
+  units: meters (east)
+  default value: '0'
+  enumerated possible values:
+  Description: Determines the baseline vector for the "init known baseline" feature.
+  Notes:
+- group: solution
+  name: known_baseline_d
+  type: double
+  units: meters (down)
+  default value: '0'
+  enumerated possible values:
+  Description: Determines the baseline vector for the "init known baseline" feature.
+  Notes:
+- group: system_info
+  name: serial_number
+  type: integer
+  units:
+  default value:
+  enumerated possible values:
+  Description: The serial number of the Piksi
+  Notes: This number should match the number on the barcode and cannot be modified
+- group: system_info
+  name: firmware_version
+  type: string
+  units:
+  default value:
+  enumerated possible values:
+  Description: Indicates the firmware version for the Local Piksi
+  Notes: 'For user generated firmware, this information will appear the same as the
+    GIT command: "git describe --dirty"'
```

Comment:
I think it's worth documenting, as it might break people's build system and might confuse an end-user (who wants to give his own version tag).  Eventually we will have people in the field with their own firmware.